### PR TITLE
Update sentence structure in the Zen of Wagtail

### PR DESCRIPTION
**Pull request by marv-tech**
The use of bullet points to emphasise the major roles of users allows the reader to skim information rapidly in a condensed style.

_Please check the following:_

-   [ ] Do the tests still pass?[^1]
-   [ ] Does the code comply with the style guide?
    -   [ ] Run `make lint` from the Wagtail root.
-   [ ] For Python changes: Have you added tests to cover the new/fixed behaviour?
-   [ ] For front-end changes: Did you test on all of Wagtail’s supported environments?[^2]
    -   [ ] **Please list the exact browser and operating system versions you tested**:
    -   [ ] **Please list which assistive technologies [^3] you tested**:
-   [ ] For new features: Has the documentation been updated accordingly?

**Please describe additional details for testing this change**.

[^1]: [Development Testing](https://docs.wagtail.org/en/latest/contributing/developing.html#testing)
[^2]: [Browser and device support](https://docs.wagtail.org/en/latest/contributing/developing.html#browser-and-device-support)
[^3]: [Accessibility Target](https://docs.wagtail.org/en/latest/contributing/developing.html#accessibility-targets)

### CODE REVIEW

File: docs/getting_started/the_zen_of_wagtail.md [hunk 1]
```diff
@@ -10,7 +10,13 @@ You can't make a beautiful website by plugging off-the-shelf modules together -
 
 ## Always wear the right hat.
 
-The key to using Wagtail effectively is to recognise that there are multiple roles involved in creating a website: the content author, site administrator, developer and designer. These may well be different people, but they don't have to be - if you're using Wagtail to build your personal blog, you'll probably find yourself hopping between those different roles. Either way, it's important to be aware of which of those hats you're wearing at any moment, and to use the right tools for that job. A content author or site administrator will do the bulk of their work through the Wagtail admin interface; a developer or designer will spend most of their time writing Python, HTML or CSS code. This is a good thing: Wagtail isn't designed to replace the job of programming. Maybe one day someone will come up with a drag-and-drop UI for building websites that's as powerful as writing code, but Wagtail is not that tool, and does not try to be.
+The key to using Wagtail effectively is to recognise that there are multiple roles involved in creating a website. Some of these roles:
```
Comment (gasman) [inline docs/getting_started/the_zen_of_wagtail.md:13]: Saying "Some of these roles" doesn't really work here - it implies that there are other roles besides the ones listed. However, the whole message here is that the user should be aware of which role they're playing - they can't really do that if they don't know what the complete list of roles is.

Comment (gasman) [reply docs/getting_started/the_zen_of_wagtail.md:13]: I'm not really a fan of the bullet points, personally - I feel it's a bit out-of-place in an essay-style article. However, as the original author of this text I'm not really the best judge of what other people find most readable - so I'm happy to go with whatever other people think.

Comment (marv-tech) [reply docs/getting_started/the_zen_of_wagtail.md:13]: Thank you for your feedback @gasman

**Comment by thibaudcolas** [comment]
I’ll close this now – agree with the feedback from Matt, and I don’t think this kind of essay is something we’d want to make it easier for people to skim through (people who still want that can stick to the headings and get some value out of the content).

**Pull request closed without merging**